commonknowledge/mapped
# WIP: Feature/map 239 auto import via webhooks cron for data sources

Still to test: Action Network

## Description
Adds auto-import webhooks to the front and back end.

## Motivation and Context
https://linear.app/commonknowledge/issue/MAP-239/auto-import-via-webhooks-cron-for-data-sources

## How Can It Be Tested?
Set up a new Airtable or Mailchimp data source. Turn on webhooks for import and update. Modify the data and verify it is (a) updated and (b) imported.

## How Will This Be Deployed?
